app/utils.py

```diff
@@ -122,3 +122,9 @@ def escape_special_characters(string):
             r'\{}'.format(special_character)
         )
     return string
+
+
+def email_address_is_nhs(email_address):
+    return email_address.lower().endswith((
+        '@nhs.uk', '@nhs.net', '.nhs.uk', '.nhs.net',
```

Comment:
Why can't this be ('nhs.uk',  'nhs.net')?

Reply:
Because that would also match `fake-nhs.net`. Although I guess it doesn’t really matter here because we’re just setting branding not ensuring the person has a government email address.

Reply:
Although thinking about it some more if we have a function called `email_address_is_nhs` then doing the more stringent check is better, in case anyone in the future was going to make use of the function elsewhere.

Reply:
fair enough